config/settings.yml

```diff
@@ -9,3 +9,5 @@ docs:
   :infra_provider: http://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#infrastructure_providers
   :network_provider: http://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#network_providers
   :physical_infra_provider: http://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#infrastructure_providers
+  :product_support_website: http://www.manageiq.org
```

Comment:
Shouldn't these be in the core repo?

Reply:
Actually, the whole `docs` section feels like it belongs in core.

Reply:
There are a number of things wrong with this, but

- Yes, I agree this entire section should live in core.
- All of the links should be https
- provider type links are fine, but then "ansible" (which should really be ansible_tower) is a provider specific link, which should be pluggable, but I'd argue that it shouldn't exist at all since we don't have any of the other providers

Reply:
I think the point of the change was to use `Settings` instead of `ui_lookup`. Anything else is optional :).
(Note it didn't actually change any of the links, feel free to update them if you know what they should be.)

As for `ansible` - maybe it should really be automation. It's used in the "no records" screen in /automation_manager/explorer (Automation > Ansible Tower > Explorer)

Reply:
(Except for `product_support_*`, none of the other keys are used anywhere outside the UI.)

Reply:
I'm having bit of a problem understanding why should this config section live in core?

Reply:
IMO it doesn't make sense to move these to the core without a core mechanism for "get me docs for this model",
and the UI won't consume it over the API before we have an API endpoint for list views with all the info the UI needs.

Reply:
I opened https://github.com/ManageIQ/manageiq/pull/20010/files and https://github.com/ManageIQ/manageiq-ui-classic/pull/6793 to move the settings.

Reply:
https://github.com/ManageIQ/manageiq-api/pull/788 is how I got here.  Which struck me as odd that UI settings are being used for the API.

Reply:
> I think the point of the change was to use Settings instead of ui_lookup

Yeah, we're not disputing the usage of Settings, we're disputing where the Settings should live.

> and the UI won't consume it over the API before we have an API endpoint for list views with all the info the UI needs. (Should we plan for pluggable provider kinds?)

Settings are already available via the API, and providers can already bring pluggable settings, though in this case they are actually provider-type which is a core construct anyway, so it doesn't matter.